integrations/spark/spark-3.1/openhouse-spark-runtime/src/main/scala/com/linkedin/openhouse/spark/sql/catalyst/parser/extensions/OpenhouseSqlExtensionsAstBuilder.scala

```diff
@@ -19,10 +19,11 @@ class OpenhouseSqlExtensionsAstBuilder (delegate: ParserInterface) extends Openh
 
   override def visitSetRetentionPolicy(ctx: SetRetentionPolicyContext): SetRetentionPolicy = {
     val tableName = typedVisit[Seq[String]](ctx.multipartIdentifier)
-    val (granularity, count) = typedVisit[(String, Int)](ctx.retentionPolicy())
+    val retentionPolicy = ctx.retentionPolicy()
```

Comment:
why is this changed required as part of this PR?

Reply:
Good catch I'll revert this change